onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -731,6 +731,19 @@ def _add_torchscript_op_call(
             tensor.name = _rename_intermediate_value(tensor.name)
         return tensors
 
+    @runtime_typing.checked
+    def fetch_opset_imports(self) -> Dict[str, int]:
+        # TODO: All local function domain versions are hardcoded as 1.
+        opset_imports: Dict[str, int] = {}
+        for subgraph in self._sub_torch_script_graphs.values():
+            opset_imports.update(subgraph.fetch_opset_imports())
```

Comment:
Do we need this to be recursive, or do we just need opsets for call the direct invocations?

Reply:
Need to be recursive, or more precisely, looking inside nested functions. This is exactly what was missing that led to the IsScalar issue.
Now there is indeed in-efficiency that can be improved. Caching lookup results could be one solution.

Reply:
Thinking about it again, caching is definitely no-go since it is more than likely to create bugs in the future.

I'll merge as is and leave it as follow-up in case people are interested to improve the efficiency.

Reply:
> This is exactly what was missing

I just realized. Because the `aten_mean_dim` function has imported the `common` domain already. That is a little counter intuitive. Although I don't see this explicitly stated in the spec. @gramalingam do you have thoughts?